p2p/kademlia/dht.go

```diff
@@ -555,15 +563,19 @@ func (dht *DHT) timers() {
 			for i := 0; i < b; i++ {
 				if time.Since(dht.ht.getRefreshTimeForBucket(i)) > dht.options.TRefresh {
 					id := dht.ht.getRandomIDFromBucket(k)
-					dht.iterate(iterateFindNode, id, nil)
+					dht.iterate(ctx, iterateFindNode, id, nil)
 				}
 			}
 
 			// Replication
-			keys := dht.store.GetAllKeysForReplication()
+			keys, err := dht.store.GetAllKeysForReplication()
+			if err != nil {
+				log.Fatal(err)
```

Comment:
```suggestion
				log.WithContext(ctx).Fatal(err)
```
If this error occurs, can `kademlia` continue to run?

Reply:
no

Reply:
fixed